schemas/app/inventory/StockTransfer.json

```diff
@@ -65,6 +65,25 @@
       "abstract": true,
       "fieldname": "backReference",
       "section": "References"
+    },
+    {
+      "fieldname": "isReturn",
+      "label": "Is Return",
+      "placeholder": "Is Return",
+      "fieldtype": "Check",
+      "default": false,
+      "section": "Return"
```

Comment:
This is kind of a redundant field. If `returnAgainst` already implies that it is a return doc `isReturn` is not needed.

And even if it's added, it can be hidden.